pkg/cmd/cli/install/install.go

```diff
@@ -130,6 +132,54 @@ func NewCommand(f client.Factory) *cobra.Command {
 	return c
 }
 
+func CheckVersion(image string) bool {
```

Comment:
The version check will return false if the version is v2.0.0 or v2.0.1. That's not right. So, please refer to Internet for some idea about version check. I believe there is a bunch utilities available.

BTW, the next version of CSI controller is just v2.0.0

Reply:
For vsphere-csi-driver, version prior to v1.0.2: v0.2.0, v0.2.1, v1.0.1, so here just check for these specific cases for simplicity.